store_data.py

```diff
@@ -0,0 +1,341 @@
+"""This file creates is for the DB-related work on the statistics."""
+
+import argparse
+import json
+
+import database_helpers as db_utils
+import MySQLdb
+import pexpect
+import shell_helpers as sh_utils
+import utils
+
+
+class DataStoreError(Exception):
+  pass
+
+
+def CreateTable(connection, table_name):
+  """This function creates the envoy_stat table on the give `connection`.
+
+  Args:
+    connection: the connection which will refer to a database in which the table
+    will be created.
+    table_name: the name of the table to be created.
+  Returns/Yields:
+    It creates a table on the `connection`.
```

Comment:
This isn't a Returns/Yields comment, it belongs above. This function doesn't return or yield any exception.